src/serialization/serialization.rs

```diff
@@ -0,0 +1,24 @@
+extern crate byteorder;
+
+#[path = "tests.rs"]
+#[cfg(test)]
+mod tests;
+
+#[path = "benchmarks.rs"]
+#[cfg(all(test, feature = "bench_private"))]
+mod benchmarks;
+
+#[path = "v2_serializer.rs"]
+mod v2_serializer;
+pub use self::v2_serializer::{V2Serializer, SerializeError};
```

Comment:
Either `SerializeError` should be placed directly in this file (if it is common to all serializer versions), or it should be renamed to `V2SerializeError`